subsys/partition_manager/pm.yml.pgps

```diff
@@ -3,4 +3,9 @@
 pgps:
   placement: {before: [tfm_storage, end]}
   size: CONFIG_NRF_CLOUD_PGPS_PARTITION_SIZE
+#ifdef CONFIG_BUILD_WITH_TFM
+  align: {start: CONFIG_NRF_SPU_FLASH_REGION_SIZE}
+#else
   align: {start: CONFIG_NRF_CLOUD_PGPS_PARTITION_ALIGN}
```

Comment:
Same comment as in https://github.com/nrfconnect/sdk-nrf/pull/7702/files#diff-1f1c4b9dd8edda0fd4c669f7f27c61c8d0dfec246cc74d853e0106dbc7112026R9